bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -673,6 +733,15 @@ impl Vm {
             inner: InnerAsyncResult(ctx),
         })
     }
+    pub async fn run_function_async2(
+        &self,
+        func_name: impl AsRef<str> + Send,
+        params: impl IntoIterator<Item = WasmValue> + Send,
+    ) -> WasmEdgeResult<Vec<WasmValue>> {
+        FiberFuture::on_fiber(|| self.run_function(func_name, params))
+            .await
+            .unwrap()
+    }
```

Comment:
Please replace the original `run_function_async` with the new `run_function_async2`, but keep using the original API name `run_function_async'. Thanks!